lib/slack.ml

```diff
@@ -246,7 +246,21 @@ let generate_status_notification (cfg : Config_t.config) (notification : status_
   let description_info =
     match description with
     | None -> None
-    | Some s -> Some (sprintf "*Description*: %s." s)
+    | Some s ->
+      let text =
+        match target_url with
+        | None -> s
+        | Some target_url ->
+          (* Specific to buildkite *)
+          let re = Re2.create_exn {|^Build #(\d+)(.*)|} in
```

Comment:
should be done at toplevel to avoid creating it many times

Reply:
https://github.com/ahrefs/monorobot/pull/146